<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="de-c-w-869" tests="2" failures="1" timestamp="7 Dec 2015 16:45:06 GMT" time="0.668" errors="0">
  <testcase name="simpleTest" time="0.608" classname="CalcDemoTest"/>
  <testcase name="targetServerReady" time="0.038" classname="CalcDemoTest">
    <failure type="java.lang.AssertionError" message="Calculator">
      <![CDATA[java.lang.AssertionError: Calculator
	at org.junit.Assert.fail(Assert.java:91)
	at org.junit.Assert.assertTrue(Assert.java:43)
	at CalcDemoTest.targetServerReady(CalcDemoTest.java:95)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- targetServerReady -->
</testsuite>
